### PR TITLE
Spp 10694 method template

### PR DESCRIPTION
**Pull request by JasonBellONS**
implementation of a method spec template

### CODE REVIEW

**Review by jhmjam — COMMENTED** [review]
Hi, I see you have a calculations section but is it also worth having a 'example'  section like we do in TAC and TPC?

File: Method_Spec_Template.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+\<This Section must contain:
+
+Similar to the previous section, A list of output data
+created by the method, including any types and where
+relevant, any examples.
+
+Again, anything note-worthy about specific outputs should
+be mentioned, (e.g if an output provides a character output
+that specifies what the method did, if it successfully
+completed or not, or if it completed, if it amended data)
```
Comment (gibbardsteve) [inline Method_Spec_Template.md:64]: minor - missing the end ">"

File: Method_Spec_Template.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+\< This Section must contain:
+A List of the different data encountered in this method,
+with an explanation of what it is
+>
+
+## 3.0 Introduction
+
+\< This section must contain:
+
+An Introduction to the methiod, describing the purpose
```
Comment (gibbardsteve) [inline Method_Spec_Template.md:20]: Typo - methiod

File: Method_Spec_Template.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+of what may be used within
+the field >
+
+Additionally include a list of any optional values
+or "variable" values, (e.g if one of two columns
+must be populated, identify which columns these are).
+
+Finally, add in anything else note-worthy about the
+inputs (e.g if an input is a "switch" value, where the
+input should be 0, 1, or 2 for instance). >
```
Comment (gibbardsteve) [inline Method_Spec_Template.md:51]: Minor missing the start "<"

File: Method_Spec_Template.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+## 7.0 Calculations
+
+\<This section must contain:
+subsections containing any calculations that the method
+does, these calculations should be written in both mathjax
+and asciimath, see the below from totals_and_components as
+an example >
+
+### 7.1 Error Detection Calculations
```
Comment (gibbardsteve) [inline Method_Spec_Template.md:101]: Might just be worth adding "Example - " to the section title to reinforce that this is example content

**Review by gibbardsteve — APPROVED** [review]

**Review by gibbardsteve — APPROVED** [review]
Looks good

**Pull request merged**